src/tiles/grid/tile-stats.ts

```diff
@@ -1,88 +1,142 @@
 
+  /** The number of times the item has been viewed */
   @property({ type: Number }) viewCount?: number;
 
+  /** The text label describing the type of views (default "all-time views") */
   @property({ type: String }) viewLabel?: string;
 
+  /** The number of times the item has been favorited */
   @property({ type: Number }) favCount?: number;
 
+  /** The number of times the item has been reviewed */
   @property({ type: Number }) commentCount?: number;
 
-  render() {
-    const formattedFavCount = formatCount(this.favCount, 'short', 'short');
-    const formattedReviewCount = formatCount(
-      this.commentCount,
-      'short',
-      'short',
-    );
-
-    const uploadsOrViewsTitle =
-      this.mediatype === 'account'
-        ? `${this.itemCount ?? 0} uploads`
-        : `${this.viewCount ?? 0} ${this.viewLabel ?? 'all-time views'}`;
+  /** Whether to show the number of TV clips in place of reviews */
+  @property({ type: Boolean }) showTvClips = false;
+
+  /** The number of times the TV item has been clipped */
+  @property({ type: Number }) tvClipCount?: number;
 
+  render() {
     return html`
       <div class="item-stats">
         <p class="sr-only">
           ${this.mediatype === 'account' ? 'Account Stats' : 'Item Stats'}
         </p>
         <ul id="stats-row">
-          <li class="col">
-            <p class="sr-only">${msg('Mediatype:')}</p>
-            <mediatype-icon .mediatype=${this.mediatype}></mediatype-icon>
-          </li>
-          <li class="col" title="${uploadsOrViewsTitle}">
-            ${this.mediatype === 'account' ? uploadIcon : viewsIcon}
-            <p class="status-text">
-              <span class="sr-only">
-                ${this.mediatype === 'account'
-                  ? msg('Uploads:')
-                  : msg('Views:')}
-              </span>
-              ${formatCount(
-                this.mediatype === 'account'
-                  ? (this.itemCount ?? 0)
-                  : (this.viewCount ?? 0),
-                'short',
-                'short',
-              )}
-            </p>
-          </li>
-          <li class="col" title="${formattedFavCount} favorites">
-            ${favoriteFilledIcon}
-            <p class="status-text">
-              <span class="sr-only">${msg('Favorites:')}</span>
-              ${formattedFavCount}
-            </p>
-          </li>
-          <li class="col reviews" title="${formattedReviewCount} reviews">
-            ${reviewsIcon}
-            <p class="status-text">
-              <span class="sr-only">${msg('Reviews:')}</span>
-              ${formattedReviewCount}
-            </p>
-          </li>
+          ${this.mediatypeIconColumnTemplate}
+          ${this.mediatype === 'account'
+            ? this.uploadsColumnTemplate
+            : this.viewsColumnTemplate}
+          ${this.favoritesColumnTemplate}
+          ${this.showTvClips
+            ? this.tvClipsColumnTemplate
+            : this.reviewsColumnTemplate}
```

Comment:
This render method was getting somewhat unwieldy given all the mixing of conditional logic and HTML throughout it. I refactored things to pull out the individual column templates and make the logic more readable.